database/migrations/2024_09_26_150430_create_pilot_passport_airfield_map_table.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration {
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up() {
+        Schema::create('pilot_passport_airfield_map', function (Blueprint $table) {
+            $table->id();
+            $table->string('airfield', length: 4);
+            $table->integer('mapped_to');
+        });
+
+        Artisan::call('db:seed', [
+            '--class' => 'PilotPassportAirfieldMapSeeder',
+            '--force' => true
+        ]);
+    }
```

Comment:
I would prefer to do this via a SQL query, however I kinda like this until the seed fails... Just putting my thoughts down, I don't think anything needs to change?

Reply:
I had to remove this - it causes the browser test to fail because seeding the empty tables is a step.  I figured that I would just manually-issue the command, but maybe that isn't the best idea. How do you suggest that I do this? Check to see if the table is empty and if so, seed it in the update routine?

Reply:
Since this is something that has to exist for the code to work, I would do this with a raw SQL query in the migration. We should be able to use `DB::statement()` or using a transaction if we think that is necessary